core/src/main/java/org/apache/druid/data/input/impl/DimensionsSpec.java

```diff
@@ -199,9 +199,6 @@ private void verify(List<SpatialDimensionSchema> spatialDimensions)
         "dimensions and dimensions exclusions cannot overlap"
     );
 
-    ParserUtils.validateFields(dimNames);
```

Comment:
it seems like this should still be there to check for true duplicates?

Reply:
This check is duplicate. It's checked again here: https://github.com/apache/incubator-druid/pull/7692/files/20ad8911652e2625f15d0c93fd95fb4b48a4672e#diff-f7e4e56e91b3b0f304d4978038fe75c9R215.

Reply:
Oh, i see, my bad :+1: